# HG changeset patch
# Parent 2897c22ff88041aa524ff35562386afad83a46a2
disable regexp debug log.  use BT regexp engine as default.

diff --git a/src/regexp.c b/src/regexp.c
index 7f5e265eb..5538a24a9 100644
--- a/src/regexp.c
+++ b/src/regexp.c
@@ -41,7 +41,7 @@
 /* Uncomment the first if you do not want to see debugging logs or files
  * related to regular expressions, even when compiling with -DDEBUG.
  * Uncomment the second to get the regexp debugging. */
-/* #undef DEBUG */
+#undef DEBUG
 /* #define DEBUG */
 
 #include "vim.h"
